### PR TITLE
Fix configmap generation when using extra option

### PR DESCRIPTION
**Pull request by ltupin**
### Issue

https://github.com/aws/eks-charts/issues/657

### Description of changes
When you want to use extra `[FILTER]` `[INPUT]` and `[OUTPUT]`  options it is confusing in comments to leave the title itself. See details and examples in https://github.com/aws/eks-charts/issues/657

That's why I want to propose a PR to indent from 4 to 8, any extra option and to remove the block configuration title `[FILTER]` `[INPUT]` and `[OUTPUT]` in comments in values file. 

The only block within I didn't try it is the PARSER so that's why I don't submit any change for this one (but maybe it is needed too).

### Checklist
- [X] Added/modified documentation as required (such as the `README.md` for modified charts)
- [X] Incremented the chart `version` in `Chart.yaml` for the modified chart(s)
- [X] Manually tested locally with helm template command and on EKS cluster
- [X] Make sure the title of the PR is a good description that can go into the release notes

### Testing

I apply it locally and it works, my `values.yaml` file:

```
filter: 
  match: "kube.*"
  kubeURL: "https://kubernetes.default.svc.cluster.local:443"
  mergeLog: "On"
  mergeLogKey: "data"
  keepLog: "On"
  k8sLoggingParser: "On"
  k8sLoggingExclude: "On"

extraFilters: |
   DNS_Retries         3
   DNS_Wait_Time       10
```

Command:
```
helm --debug template -s templates/configmap.yaml aws-for-fluent-bit  aws-for-fluent-bit -f values.yaml
```

Output:
```
    [FILTER]
        Name                kubernetes
        Match               kube.*
        Kube_URL            https://kubernetes.default.svc.cluster.local:443
        Merge_Log           On
        Merge_Log_Key       data
        Keep_Log            On
        K8S-Logging.Parser  On
        K8S-Logging.Exclude On
        DNS_Retries         3
        DNS_Wait_Time       10
```

**I deploy it on my EKS clusters and it works.**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by PettitWesley — APPROVED** [review]

**Comment by ltupin** [comment]
@PettitWesley thx a lot but dont approve it right now please because I discover another issue. The indent seems not working ! 

I'll have a look deeper on it tomorrow. 

Is it possible to modify a PR or I would have to submit a new one ?

**Comment by PettitWesley** [comment]
@ltupin You can modify a PR yes (preferably by force-pushing your branch to a new commit IMO)... but if its not ready... please convert it to a draft and only make it a real PR once you have fully tested and are truly ready...

**Comment by ltupin** [comment]
Done, I have converted it to draft, sorry for noise,  I will resubmit it when fully tested. Sorry again @PettitWesley.

**Comment by ltupin** [comment]
@PettitWesley finally after tested on local and real EKS kube cluster I can confirm it it ok. I was afraid but the problem was on my environment not into my modifications.

Just need to increase the chart version and you can merge it.

**Comment by ltupin** [comment]
@PettitWesley @hossain-rayhan could you please check it ?

**Review by PettitWesley — APPROVED** [review]

File: stable/aws-for-fluent-bit/values.yaml [hunk 1]
```diff
@@ -108,7 +105,6 @@ elasticsearch:
   replaceDots: "On"
 
 # extraOutputs: |
-#   [OUTPUT]
```
Comment (hossain-rayhan) [inline stable/aws-for-fluent-bit/values.yaml:111]: Why removing this single line comment? Either remove the whole output section or don't touch them. Same for above input and filter section.

**Review by hossain-rayhan — APPROVED** [review]
Added a minor comment. Otherwise, looks good.

**Comment by Anindya-Banerjee** [comment]
Any estimate on when this PR can be merged? We are in desperate need of this fix.

File: stable/aws-for-fluent-bit/values.yaml (hunk 1 above)
Comment (ltupin) [reply stable/aws-for-fluent-bit/values.yaml:111]: Yes @PettitWesley, as I explained in my introduction above `and to remove the block configuration title [FILTER] [INPUT] and [OUTPUT] in comments in values file.` The reason is it makes people confused and if you keep two blocks, it won't work...

**Comment by ltupin** [comment]
> @ltupin @Anindya-Banerjee I can't merge it since there are conflicts :/

The conflict is "just" the version because since I did it, it evolve:
`
<<<<<<< master
appVersion: 2.13.0
=======
appVersion: 2.21.5
>>>>>>> master
`

**Comment by ltupin** [comment]
"et voilà" @PettitWesley :-)
Could you please push it ?

**Comment by ldcorentin** [comment]
@PettitWesley I have been following this issue for a while, I would be very grateful if it could be merged :-)

**Comment by PettitWesley** [comment]
@ltupin @ldcorentin Cheers :)

**Pull request merged**